examples/website/i3s/app.js

```diff
@@ -39,6 +39,17 @@ const INITIAL_VIEW_STATE = {
   zoom: 14.5
 };
 
+// https://github.com/tilezen/joerd/blob/master/docs/use-service.md#additional-amazon-s3-endpoints
+const MAPZEN_TERRAIN_IMAGE = `https://s3.amazonaws.com/elevation-tiles-prod/terrarium/{z}/{x}/{y}.png`
+const ARCGIS_STREET_MAP_SURFACE_IMAGE = 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}';
+const MAPZEN_ELEVATION_DECODER = {
```

Comment:
"Decoder" makes it sound like an instance of a class

```suggestion
const MAPZEN_ELEVATION_DECODE_PARAMETERS = {
```